config/panda_arm.xacro

```diff
@@ -7,7 +7,6 @@
         <!--CHAINS: When a chain is specified, all the links along the chain (including endpoints) are included in the group. Additionally, all the joints that are parents to included links are also included. This means that joints along the chain and the parent joint of the base link are included in the group-->
         <!--SUBGROUPS: Groups can also be formed by referencing to already defined group names-->
         <group name="panda_arm">
-            <joint name="virtual_joint"/>
```

Comment:
Obviously, for Gazebo it's important to not list the virtual joint here. This should be fixed in MSA as well. Could you please add a TODO?